qiskit_ibm/experiment/ibm_experiment_service.py

```diff
@@ -506,6 +518,7 @@ def _api_to_experiment_data(
             "experiment_type": raw_data['type'],
             "backend": backend,
             "experiment_id": raw_data['uuid'],
+            "parent_experiment_uuid": raw_data.get('parent_experiment_uuid', None),
```

Comment:
```suggestion
            "parent_id": raw_data.get('parent_experiment_uuid', None),
```
This method converts API data (e.g. "parent_experiment_uuid") to experiment data (e.g. "parent_id")